platform-tests/src/test/java/org/junit/AutomaticModuleNameTests.java

```diff
@@ -59,6 +59,7 @@ private static String version(String module) {
 		throw new AssertionError("module name is unknown: " + module);
 	}
 
+	@SuppressWarnings("FilesLinesLeak") // stream closed indirectly by JUnit 5
```

Comment:
For further details, see also https://stackoverflow.com/questions/22921623/closing-streams-in-the-middle-of-pipelines